model/cluster_utility.go

```diff
@@ -80,7 +80,7 @@ var DefaultUtilityVersions map[string]*HelmUtilityVersion = map[string]*HelmUtil
 	// PromtailCanonicalName defines the default version and values path for the Helm chart
 	PromtailCanonicalName: {Chart: "6.2.2", ValuesPath: ""},
 	// RtcdCanonicalName defines the default version and values path for the Helm chart
-	RtcdCanonicalName: {Chart: "1.1.0", ValuesPath: ""},
+	RtcdCanonicalName: {Chart: "1.3.0", ValuesPath: ""},
```

Comment:
Did you mean to bump this? The test failure in CI was triggered from this change.

Reply:
I will search why, yes it should be bumped from previous changes